### ISSUE TITLE
Document a backup mechansim

### ISSUE THREAD

**Issue by remay** [remay/tmf-hudl#8]
It would be good to have a documented (and easy?) backup mechanism for people to allow then to go back if they have problems with flashing.

Possibly based on rkdumper? https://forum.xda-developers.com/general/rooting-roms/tool-rkdumper-utility-backup-firmware-t2915363

Perhaps package up a script based on rkDumper and imgerpackrk that can create an image that could be flashed back?

**Comment by RayyanAnsari** [comment]
For now I've just backed up every partition except userdata.

**Comment by remay** [comment]
Yes, backing up all partitions works.  This topic was intended to be more about offering a backup solution for less technically minded people.
